source/basic.tex

```diff
@@ -1106,7 +1106,7 @@
 \begin{codeblock}
 typedef unsigned char T;
 template<class T
-  = T               // lookup finds the typedef name of \tcode{unsigned char}
+  = T               // lookup finds the \grammarterm{typedef-name} of \tcode{unsigned char}
```

Comment:
"of" seems awkward here. Is there a term like "resolving to" that we use for this?

Reply:
removed "of unsigned char", which doesn't seem to add much in this example

Reply:
I still see the "of unsigned char" here.

Reply:
Should be fixed now.